src/Nethermind/Nethermind.Network.Stats/Model/Node.cs

```diff
@@ -73,6 +73,7 @@ public string ClientId
 
         public string EthDetails { get; set; }
         public long CurrentReputation { get; set; }
+        public string Enr { get; set; }
```

Comment:
Can be marked with ?, as it's not always set in case of discv4. Btw can we request enr even if enrsequence is not available